test/network_test.go

```diff
@@ -0,0 +1,15 @@
+package test
+
+func (s *CLITestSuite) TestNetworkDescribe() {
+	tests := []CLITest{
+		{args: "network describe n-abcde1", fixture: "network/describe.golden"},
+		{args: "network describe n-abcde1 --output json", fixture: "network/describe-json.golden"},
```

Comment:
Feel free to delete this, since we already are testing serialized output with the YAML test below.
```suggestion
```

Reply:
Deleted!